src/app/main.tsx

```diff
@@ -11,6 +12,7 @@ ReactDOM.render(
       <Route path="/" element={<LoginPage />} />
       <Route path="/Welcome" element={<WelcomePage />} />
       <Route path="/Search" element={<SearchPage />} />
+      <Route path="/Result" element={<SearchResultPage />} />
```

Comment:
I'd usually not use capital letters in the paths because they could be case sensitive.
For more info, take a look here: https://webmasters.stackexchange.com/questions/90339/why-are-urls-case-sensitive

```suggestion
      <Route path="/welcome" element={<WelcomePage />} />
      <Route path="/search" element={<SearchPage />} />
      <Route path="/result" element={<SearchResultPage />} />
```